src/components/Colors/UIDocCard.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import Card from "../Card/Card";
+
+interface UIDocCardProps {
+    /** backgroundColor of the doc card */
+    backgroundColor: string;
+}
+
+export default function UIDocCard(
```

Comment:
Is this meant just to be used for Storybook?

Reply:
Yeah, it is... should I maybe add Storybook to the name of it so that's clear?

Reply:
No. Just a comment above the function is enough.
```
/**
 * UIDocCard
 * Component only used for Storybook....
 */
export default function UIDocCard( .....
```